### DISCUSSION TITLE
Installation problem NET 8 WinForm

### DISCUSSION THREAD

**Discussion by meisenbart77** [cefsharp/CefSharp#4800] [Q&A]
Hello,

I have written a Vb.net programme using the CefSharp.WinForms.NetCore.
The target framework is NET 8

In the Visual Studio environment the programme runs without problems, but I am not able to distribute the programme.

I use Advanced Installer 21.7.1 to create the installation programme.

I add these files manually:
CefSharp.BrowserSubprocess.Core.dll
CefSharp.BrowserSubprocess.dll
CefSharp.BrowserSubprocess.exe

After I have installed the programme with the installation program. I cannot open it. After the double click happens: NOTHING

What could be the problem?

**Comment by rafaelvictal** [comment]
Sounds like a startup issue. Does your Main sub is on a form or another file?
You could try adding a try-catch block to the Main sub and log the error on a file or the Event Viewer.
It will probably give an idea of what is wrong.

`EventLog.WriteEntry("MyApp", ex.Message);`

**Comment by amaitland** [comment]
> I add these files manually:
> CefSharp.BrowserSubprocess.Core.dll
> CefSharp.BrowserSubprocess.dll
> CefSharp.BrowserSubprocess.exe

Make sure you've included all the unmanaged dependencies. See https://github.com/cefsharp/CefSharp/wiki/Output-files-description-table-%28Redistribution%29 

Also make sure you have `Visual C++` installed or included with your installer. https://github.com/cefsharp/CefSharp/wiki/Output-files-description-table-%28Redistribution%29#visual-c
